common/errors.js

```diff
@@ -142,12 +142,16 @@
                 window.location.reload();
             }; 
             if (exceptionFlag || window.location.pathname.indexOf('/search/') != -1 || window.location.pathname.indexOf('/viewer/') != -1) return;
-
+            
             if (ERMrest && exception instanceof ERMrest.UnauthorizedError || exception.code == errorNames.unauthorized) {
                 Session.loginInAModal(reloadCb);
             }
             else if (exception.code && exception.code == errorNames.multipleRecords){
                 errorPopup(messageMap.multipleDataMessage, messageMap.multipleDataErrorCode,"Recordset ", exception.redirectUrl);
+            }
+            
+            else if (ERMrest && exception instanceof ERMrest.UnauthorizedError || exception.code == errorNames.forbidden) {
```

Comment:
If you refactor those errors to use ERMrestjs, then this check for `expection.code == errorNames.forbidden` won't be necessary. Although some other piece of code might still be using this error name. So you can just add a TODO that we should remove it when we get to refactor error classes in chaise.